src/eslintrc.json

```
@@ -0,0 +1,27 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "prettier"
+  ],
+  "plugins": ["react", "react-hooks"],
+  "parserOptions": {
+    "ecmaVersion": 2022,
```

Nit: 이건 항상 최신으로 유지할 수 있도록 아래와 같이 하는게 좋을 것 같습니다.
```suggestion
    "ecmaVersion": "latest",
```